base/cvd/cuttlefish/host/commands/cvd/cli/commands/host_tool_target.cpp

```diff
@@ -89,14 +88,29 @@ Result<std::string> HostToolTarget::GetRestartBinName() const {
   return CF_EXPECT(GetBinName({"restart_cvd"}));
 }
 
+Result<std::string> HostToolTarget::GetRestartBinPath() const {
+  return fmt::format("{}/bin/{}", artifacts_path_,
+                     CF_EXPECT(GetRestartBinName()));
+}
+
 Result<std::string> HostToolTarget::GetPowerwashBinName() const {
   return CF_EXPECT(GetBinName({"powerwash_cvd"}));
 }
 
+Result<std::string> HostToolTarget::GetPowerwashBinPath() const {
+  return fmt::format("{}/bin/{}", artifacts_path_,
+                     CF_EXPECT(GetPowerwashBinName()));
+}
+
 Result<std::string> HostToolTarget::GetPowerBtnBinName() const {
   return CF_EXPECT(GetBinName({"powerbtn_cvd"}));
 }
 
+Result<std::string> HostToolTarget::GetPowerBtnBinPath() const {
+  return fmt::format("{}/bin/{}", artifacts_path_,
+                     CF_EXPECT(GetPowerBtnBinName()));
+}
+
```

Comment:
What is the distinction between functions that use `artifacts_path_` and functions that use `GetBinName()`?